### PR TITLE
Beaglebone Blue: Further changes from BKung

### PR DESCRIPTION
**Pull request by hamishwillee**
These are @bkueng changes from https://github.com/PX4/px4_user_guide/pull/678/commits/5284c45f24bbeff59dd7326745c6a6e0f02c0b88 plus some minor modifications from me.

Fixes #668

### CODE REVIEW

File: en/flight_controller/beaglebone_blue.md [hunk 1]
```diff
@@ -28,40 +28,51 @@ The recommended way to build PX4 for *BeagleBone Blue* is to compile on a develo
...
+   For *rsync* over SSH with key authentication, follow the steps here (on the development machine):
+   1. Generate an SSH key if you have not previously done so: 
+      ```
+      ssh-keygen -t rsa
+      ```
+      1. ENTER //no passphrase
+      1. ENTER
+      1. ENTER
+   1. Define the BeagleBone Blue board as `beaglebone` in **/etc/hosts**:
+      ```
```
Comment (hamishwillee) [inline en/flight_controller/beaglebone_blue.md]: @bkueng In your version there were a number of steps here - but I think this was intended to be part of this step.

**Comment by hamishwillee** [comment]
@bkueng I merged your changes manually - look good thanks. Just one confirmation above: https://github.com/PX4/px4_user_guide/pull/679#discussion_r390045677

File: en/flight_controller/beaglebone_blue.md (hunk 1 above)
Comment (bkueng) [reply en/flight_controller/beaglebone_blue.md]: No, `Define the BeagleBone Blue board as `beaglebone` in **/etc/hosts**` is its own (optional) step. `ssh-copy-id root@beaglebone` Is required for password-less ssh access.

Comment (hamishwillee) [reply en/flight_controller/beaglebone_blue.md]: @bkueng I'm not clear on the change - including whether the prompt to trust newting. Please confirm this is correct, or make the change yourself in source :-)

   1. Define the BeagleBone Blue board as `beaglebone` in **/etc/hosts**:
      ```
      ssh-copy-id root@beaglebone
      ```
   1. Alternatively you can use the beaglebone's IP for password-less SSH access:
      ```
      ssh-copy-id root@<IP>
      ```
      * When prompted if you trust: yes

Comment (bkueng) [reply en/flight_controller/beaglebone_blue.md]: Like this (you'll be prompted when executing `ssh-copy-id`):

   1. Define the BeagleBone Blue board as `beaglebone` in **/etc/hosts** and copy the public SSH key to the board for password-less SSH access:
      ```
      ssh-copy-id root@beaglebone
      ```
   1. Alternatively you can use the beaglebone's IP directly:
      ```
      ssh-copy-id root@<IP>
      ```
   1. When prompted if you trust: yes
   1. Enter root password

Comment (hamishwillee) [reply en/flight_controller/beaglebone_blue.md]: Thanks. Copied in https://github.com/PX4/px4_user_guide/commit/1f4471ec1328041560af8cb6e91bd2a030912a72

**Pull request merged**